server/channels/app/user.go

```diff
@@ -1045,10 +1045,10 @@ func (a *App) UpdateActive(c request.CTX, user *model.User, active bool) (*model
 
 	a.sendUpdatedUserEvent(*ruser)
 
-	if pluginsEnvironment := a.GetPluginsEnvironment(); pluginsEnvironment != nil && !active && user.DeleteAt != 0 {
+	if !active && user.DeleteAt != 0 {
```

Comment:
Why we gave up on `pluginsEnvironment` check here?

Reply:
`RunMultiHook` does check if the a pluginsEnvironment exists. This means that with the new code, we do spawn a go routine, even if plugins are disabled. Given that this is consistent with the rest of the code base (e.g. https://github.com/mattermost/mattermost/blob/773ad33369ac4f227a4fff959b5a02c012655d30/server/channels/app/platform/web_conn.go#L269-L275) I think it's fine.